web/src/constants/errors/server-errors.ts

```diff
@@ -14,7 +14,12 @@
  * limitations under the License.
  */
 
+import {MAX_NUM_COMMITS} from 'constants/customer';
+
 export const GENERIC_ERROR =
   'Oops, something went wrong. Please try again later!';
+
 export const NO_MERCHANT_WITH_FIREBASE_UID =
   'There is no user registered with Firebase User ID';
```

Comment:
nit: add . at the end

Reply:
@ifhuang Oh this one is not added by me! @karenfrilya97 added it when she was doing the sign-in for merchant and she left out the `.` because she wanted to append the offending Firebase uid to the error message haha